src/python/pants/pantsd/pants_daemon.py

```diff
@@ -312,6 +312,17 @@ def _run_services(self, services):
       self._logger.info('starting service {}'.format(service))
       try:
         service_thread.start()
+        if isinstance(service, SchedulerService):
```

Comment:
how would you feel about moving the pidfile determination and passing into a `PantsDaemon` method that would then be called in `PantsDaemon._setup_services()` to pass onto the `SchedulerService` constructor - and then having `SchedulerService` apply the watch subscription in `SchedulerService.setup()` when it sets up the all files handler?

this would more closely align to the existing `PantsService` lifecycle.